emnify/constants.py

```diff
@@ -39,13 +40,15 @@ class RequestDefaultHeadersKeys(ExtendedEnum):
     AUTHORIZATION = 'authorization'
     XEmnOriginApp = 'x-emn-origin-app'
     XEmnOriginAppVersion = 'x-emn-origin-app-version'
+    USER_AGENT = 'User-Agent'
 
 
 class RequestDefaultHeadersValues(ExtendedEnum):
     APPLICATION_JSON = 'application/json'
     BEARER_TOKEN = 'Bearer {}'
     PYTHONSDK_VERSION = EMNIFY_PACKAGE_VERSION
     PYTHONSDK = 'PythonSDK'
```

Comment:
```suggestion
    PYTHONSDK = 'emnify PythonSDK'
```

Reply:
wondering what a good pattern would be.. assuming we have multiple SDKs following that schema, I guess a leading `emnifySDK` and only afterwards (somewhere) the language would make sense to filter more easily.

Reply:
You can find current example [here](https://github.com/emnify/emnify-sdk-python/pull/253/files#diff-1ec6ad7a01cbbfe90ad8531ae82907860658905459cd44a66f5405b4e648b661R10), it aligns with standaard, and includes variety of relevant info.